apps/src/sites/code.org/pages/views/swiper_page_self_paced_pl.js

```diff
@@ -9,6 +9,7 @@ import {register} from 'swiper/element/bundle';
 register();
 
 const swipers = [
+  document.querySelector('swiper-container.swiper-00'),
```

Comment:
This is a good workaround! A few PRs ago I started naming each carousel with a descriptive class name instead of numerically. If you want to do that here feel free, but def not blocking.

Reply:
haha I almost renamed them all to descriptive names then didn't. I might do that, though, as I should be able to fit that in its own commit for review.

I'll retag you when I do!